src/common/translations/localize.ts

```diff
@@ -43,16 +43,19 @@ export const computeLocalize = (
   cache._localizationCache = {};
 
   return (key, ...args) => {
-    if (!key || !resources || !language || !resources[language]) {
+    if (!key) {
       return "";
     }
+    if (!resources || !language || !resources[language]) {
```

Comment:
We shouldn't add this. If this is desired at a location, we should add it to that specific location.

Reply:
By putting it this high in the stack, we make the choice for everyone.

Reply:
If for some reason, the key cannot be found in resource file, or resource file itself is not ready, the key itself is better than empty string. UI should be still usable than an blank page.

Return key:
![image](https://user-images.githubusercontent.com/7366491/53065925-8e577580-3482-11e9-8d7d-cfd959beb8fb.png)

Return ""
![image](https://user-images.githubusercontent.com/7366491/53066002-d1194d80-3482-11e9-8e7e-f669f17779f9.png)

Reply:
Maybe not? Look at the panel title and media_player state, they seem have default fall back value, if localize() returned "", it will display a meaningful text.

Reply:
if a user sees blank text, they might refresh faster, solving it properly. Now we are implementing a system to make the system semi-usable when it's in a broken state?